install/error-handling.sh

```diff
@@ -20,7 +26,7 @@ function send_envelope() {
   # If we haven't sent the envelope file, make it and send to Sentry
   # The format is documented at https://develop.sentry.dev/sdk/envelopes/
   # Grab length of log file, needed for the envelope header to send an attachment
-  local file_length=$(stat -f %z < "$basedir/$log_file")
+  local file_length=$(wc -c < "$basedir/$log_file" | awk '{print $1}')
```

Comment:
turns out we do need `wc -c`

https://www.cyberciti.biz/faq/howto-bash-check-file-size-in-linux-unix-scripting/

The syntax for stat is different for different platforms and was failing during our unit tests. I checked to make sure that wc -c works

Reply:
Did you test with logs that have the ▶️ symbol?

Reply:
yeah I did, works locally for me